src/app.js

```diff
@@ -1 +1,28 @@
-// write code here
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const fun = (firsts, seconds) => {
+  if (!fs.existsSync(firsts)) {
```

Comment:
This line checks if the source file exists, which is good. However, it might be beneficial to also check if the source is indeed a file and not a directory using `fs.statSync(firsts).isFile()`.